### PR TITLE
MK-6 Update Update functions

### PR DESCRIPTION
**Pull request by KaskMartin**

### CODE REVIEW

File: TaisKohtApi/Controllers/api/IngredientsController.cs [hunk 1]
```diff
@@ -114,14 +114,14 @@ public IActionResult Post([FromBody]PostIngredientDTO ingredientDTO)
...
-        /// <response code="204">Ingredient was successfully updated, no content to be returned</response>
+        /// <response code="200">Ingredient was successfully updated, updated Ingredient to be returned</response>
         /// <response code="400">Faulty request, please review ID and content body</response>
         /// <response code="429">Too many requests</response>
         /// <response code="500">Internal error, unable to process request</response>
         // PUT: api/v1/Ingredients/5
         [Authorize(Roles = "admin, normalUser, premiumUser")]
         [HttpPut("{id}")]
-        [ProducesResponseType(204)]
+        [ProducesResponseType(typeof(IngredientDTO), 200)]
```
Comment (sikumiku) [inline TaisKohtApi/Controllers/api/IngredientsController.cs]: PUT ei tagasta 200, normaalne response on 204 (operation completed successfully), kui tahta tagastada objekti, võiks olla 201 at the very least. 
**https://stackoverflow.com/questions/797834/should-a-restful-put-operation-return-something?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa**

File: TaisKohtApi/Controllers/api/MenusController.cs [hunk 2]
```diff
@@ -118,14 +118,14 @@ public IActionResult Post([FromBody]PostMenuDTO menuDTO)
...
-        /// <response code="204">Menu was successfully updated, no content to be returned</response>
+        /// <response code="200">Menu was successfully updated, updated Menu to be returned</response>
         /// <response code="400">Faulty request, please review ID and content body</response>
         /// <response code="429">Too many requests</response>
         /// <response code="500">Internal error, unable to process request</response>
         // PUT: api/v1/Menus/5
         [Authorize(Roles = "admin, normalUser, premiumUser")]
         [HttpPut("{id}")]
-        [ProducesResponseType(204)]
+        [ProducesResponseType(typeof(MenuDTO), 200)]
```
Comment (sikumiku) [inline TaisKohtApi/Controllers/api/MenusController.cs]: Sama mis above

File: TaisKohtApi/Controllers/api/PromotionsController.cs [hunk 3]
```diff
@@ -118,7 +118,7 @@ public IActionResult Post([FromBody]PromotionDTO promotionDTO)
         ///     }
         ///
         /// </remarks>
-        /// <response code="204">Promotion was successfully updated, no content to be returned</response>
+        /// <response code="200">Promotion was successfully updated, updated Promotion to be returned</response>
```
Comment (sikumiku) [inline TaisKohtApi/Controllers/api/PromotionsController.cs]: Sama, mis above

File: TaisKohtApi/Controllers/api/RatingLogsController.cs [hunk 4]
```diff
@@ -113,7 +113,7 @@ public IActionResult PostRestaurantRatingLog([FromBody]RatingLogForEntityDTO rat
         ///     }
         ///
         /// </remarks>
-        /// <response code="204">Rating was successfully updated, no content to be returned</response>
+        /// <response code="200">Rating was successfully updated, updated Rating to be returned</response>
```
Comment (sikumiku) [inline TaisKohtApi/Controllers/api/RatingLogsController.cs]: Sama, mis above

File: TaisKohtApi/Controllers/api/RestaurantsController.cs [hunk 5]
```diff
@@ -166,33 +166,35 @@ public IActionResult Post([FromBody]RestaurantDTO restaurantDTO)
         ///     }
         ///
         /// </remarks>
-        /// <response code="204">Restaurant was successfully updated, no content to be returned</response>
+        /// <response code="200">Restaurant was successfully updated, updated Restaurant to be returned</response>
```
Comment (sikumiku) [inline TaisKohtApi/Controllers/api/RestaurantsController.cs]: Sama, mis above

File: TaisKohtApi/Controllers/api/IngredientsController.cs (hunk 1 above)
Comment (KaskMartin) [reply TaisKohtApi/Controllers/api/IngredientsController.cs]: https://httpstatuses.com/204 -> no content aga me tagastame uue objekti

File: TaisKohtApi/Controllers/api/MenusController.cs (hunk 2 above)
Comment (KaskMartin) [reply TaisKohtApi/Controllers/api/MenusController.cs]: https://httpstatuses.com/204

File: TaisKohtApi/Controllers/api/PromotionsController.cs (hunk 3 above)
Comment (KaskMartin) [reply TaisKohtApi/Controllers/api/PromotionsController.cs]: https://httpstatuses.com/204

File: TaisKohtApi/Controllers/api/RatingLogsController.cs (hunk 4 above)
Comment (KaskMartin) [reply TaisKohtApi/Controllers/api/RatingLogsController.cs]: https://httpstatuses.com/204

File: TaisKohtApi/Controllers/api/RestaurantsController.cs (hunk 5 above)
Comment (KaskMartin) [reply TaisKohtApi/Controllers/api/RestaurantsController.cs]: https://httpstatuses.com/204

**Pull request merged**